diff --git a/linker/linker_aarch64.ld b/linker/linker_aarch64.ld
index 47bd13142..77883a1a6 100644
--- a/linker/linker_aarch64.ld
+++ b/linker/linker_aarch64.ld
@@ -41,7 +41,9 @@ SECTIONS
     .boot_stack : ALIGN(4K)
     {
         _stack_start_low = .;
+        KEEP(*(.marker.stack.before))
         *(.boot_stack)
+        KEEP(*(.marker.stack.after))
         _stack_end_low = .;
     } :boot_data
 
